lib/galaxy_test/api/test_group_roles.py

```diff
@@ -67,7 +67,7 @@ def test_update(self):
         self._assert_status_code_is_ok(update_response)
         group_role = update_response.json()
         self._assert_valid_group_role(group_role, assert_id=encoded_role_id)
-        assert group_role["url"] == f"/api/groups/{encoded_group_id}/roles/{encoded_role_id}"
+        assert (group_role["url"] == f"/api/groups/{encoded_group_id}/roles/{encoded_role_id}" or group_role["url"] == "*deprecated attribute not filled in by FastAPI server*")
```

Comment:
This may sound like a good idea in principle, but we are coupling the deprecation message with the test and if the message changes at some point we will have to change it everywhere, so maybe we can relax here the test and just remove the `assert group_role["url"] == f"/api/groups/{encoded_group_id}/roles/{encoded_role_id}"`, that would be my quick&dirty advice, but maybe someone has a better idea? :)

Reply:
The message itself is generic and can be applied to _any_ deprecated attribute. Maybe just declare it as a constant in `webapps/galaxy/api/common.py` (or a function that takes the attr name, etc.)? That is, if we _do_ want to test this endpoint.

Reply:
Sounds good, should I add that in this PR?